service/frontend/workflowHandler.go

```diff
@@ -1942,7 +1942,7 @@ func (wh *WorkflowHandler) ListClosedWorkflowExecutions(ctx context.Context,
 	} else if listRequest.StatusFilter != nil {
 		persistenceResp, err = wh.visibitiltyMgr.ListClosedWorkflowExecutionsByStatus(&persistence.ListClosedWorkflowExecutionsByStatusRequest{
 			ListWorkflowExecutionsRequest: baseReq,
-			Status:                        listRequest.GetStatusFilter(),
+			Status: listRequest.GetStatusFilter(),
```

Comment:
need `make fmt`? Or mismatch go tool version lead to inconsistent fmt issue?

Reply:
yea I am figuring it out